bundles/org.openhab.binding.chromecast/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -200,13 +214,23 @@
 		<item-type>Number:Time</item-type>
 		<label>Current Time</label>
 		<description>Current time of currently playing media</description>
+		<category>Time</category>
+		<tags>
+			<tag>Status</tag>
+			<tag>Duration</tag>
+		</tags>
 		<state readOnly="true" pattern="%.0f %unit%"/>
 	</channel-type>
 
 	<channel-type id="duration" advanced="true">
 		<item-type>Number:Time</item-type>
 		<label>Duration</label>
 		<description>Length of currently playing media</description>
+		<category>Time</category>
+		<tags>
+			<tag>Point</tag>
```

Comment:
I'm not sure if "raw" point tag makes sense in combination with a property tag like below. I guess it might viloate rule 1 by not being absolutely obvious. I would need @lolodomo's advise here.